modules/assim.batch/R/helpers.R

```diff
@@ -0,0 +1,30 @@
+stop <- function(...) {
```

Comment:
Ah I see. Makes sense for RTM and perhaps the other stand-alone tools too. @mdietze should we make it common practice for tools that are run separate from the main workflow to minimize their dependency on PEcAn packages for things like the logging functions?